app/graphql/types/query_type.rb

```diff
@@ -122,7 +137,7 @@ def construct_query(searchterm, facets)
         query[:language] = facets[:languages]
         query[:literary_form] = facets[:literary_form]
         query[:source] = facets[:source] if facets[:source] != 'All'
-        query[:subject] = facets[:subjects]
+        query[:subjects_facet] = facets[:subjects]
```

Comment:
I may have missed this discussion, but it looks like this change is being made to the v1 side of the flipflop? It doesn't look like this is matched by anything in the search model, so I'm not sure if I'm missing something.

Reply:
Yeah I think you are correct. As this should still be using ElasticSearch on the v1 side of the if/else this should remain the old name. I think.